From e5cbb7e3d3d8214eecfc1c48f3d1089d3097baf6 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Tue, 9 Aug 2016 17:51:04 +0800
Subject: [PATCH 1124/5242] MLK-13022-3 arm: dts: imx6ull-14x14-evk-emmc.dts:
 set usdhc2 per clock to 132MHz

commit  08eea097dcaf1d3e8600f65f78e3cafc1d56599d from
https://source.codeaurora.org/external/imx/linux-imx.git

For imx6ull-14x14-evk emmc rework board, usdhc2 HS200 I/O timing is
not stable @176MHz, for write operation, crc error shows up. So this
patch change the frequence to 132MHz.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-evk-emmc.dts |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-evk-emmc.dts b/arch/arm/boot/dts/imx6ull-14x14-evk-emmc.dts
index 5097ec8..bf8db20 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-evk-emmc.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-evk-emmc.dts
@@ -14,8 +14,9 @@
 	pinctrl-1 = <&pinctrl_usdhc2_8bit_100mhz>;
 	pinctrl-2 = <&pinctrl_usdhc2_8bit_200mhz>;
 	assigned-clocks = <&clks IMX6UL_CLK_USDHC2_SEL>, <&clks IMX6UL_CLK_USDHC2>;
-	assigned-clock-parents = <&clks IMX6UL_CLK_PLL2_PFD0>;
-	assigned-clock-rates = <0>, <176000000>;
+	assigned-clock-parents = <&clks IMX6UL_CLK_PLL2_PFD2>;
+	assigned-clock-rates = <0>, <396000000>;
+	max-frequency = <132000000>;
 	bus-width = <8>;
 	non-removable;
 	status = "okay";
-- 
1.7.9.5

